packages/orm/src/client/crud/operations/exists.ts

```diff
@@ -0,0 +1,12 @@
+import type { SchemaDef } from '../../../schema';
+import { BaseOperationHandler } from './base';
+
+export class ExistsOperationHandler<Schema extends SchemaDef> extends BaseOperationHandler<Schema> {
+    async handle(_operation: 'exists', args: unknown): Promise<unknown> {
+        // normalize args to strip `undefined` fields
```

Comment:
I'm all for comments but probably not super helpful in this case, just something to potentially clean up.